### ISSUE TITLE
Away in personal emergency - mum was in ICU on ventilator life support

### ISSUE THREAD

**Issue by kensoh** [tebelorg/RPA-Python#144]

**Comment by pythiantech** [comment]
@kensoh my prayers and best wishes with you and your family

**Comment by kensoh** [comment]
Thank you @pythiantech, medical team has stopped sedation and we are praying for my mum to smoothly regain consciousness and to start breathing on her own soon 🙏🏻

**Comment by bitknol** [comment]
My prayers with you and family. Hope she gets well soon.

**Comment by ongkokl** [comment]
@kensoh 

Wish your mum a speedy recovery 🙏🏼🙏🏼🙏🏼

**Comment by kensoh** [comment]
Thank you @bitknol and @ongkokl she is starting to regain consciousness. Still on ventilator life support. Medical team has started treatment on her microscopic polyangiitis condition and we are praying for improvement in her lungs to be able to breathe on her own soon.

**Comment by kensoh** [comment]
Mum transitted to HDU (high dependency unit) on oxygen support without ventilator.

**Comment by pythiantech** [comment]
That's great news! Wish her a speedy and full recovery.

**Comment by kensoh** [comment]
Thanks @pythiantech ! We'll need all the medical care, prayers and well wishes to pull through this ordeal.

Mum has now transitted out of HDU, praying for her recovery of kidneys, and able to wean off oxygen support.

**Comment by oualidn7** [comment]
Hope your mum gets well soon.

**Comment by fuzaro** [comment]
Hi @kensoh ,
I knew this project recently and that was a open door to know more about a person behind the code.
I  hope that all things are going well with you and your mum.

**Comment by rafskov** [comment]
Hi @kensoh sending thoughts and prayers for you and the family all the way from San Francisco!

**Comment by pauljohnfox** [comment]
Hello: 

Hope your mother has pulled through. Hope you're okay too.

**Comment by kensoh** [comment]
Hi @oualidn7 @fuzaro @rafskov @pauljohnfox thanks for dropping a message, I really appreciate it.

Wow, it has been 4 months.. Mum has transitted to general ward, and seems to be dialysis-free now. We are praying for her kidney function to maintain, and to be able to breathe without oxygen mask.

I'm mostly exhausted, but am counting my blessing to be able to be with family and also take care of my toddler until she reaches the age to enter childcare. Then I'll head back to work at AI Singapore.

**Comment by pauljohnfox** [comment]
> Hi @oualidn7 @fuzaro @rafskov @pauljohnfox thanks for dropping a message, I really appreciate it.
> 
> Wow, it has been 4 months.. Mum has transitted to general ward, and seems to be dialysis-free now. We are praying for her kidney function to maintain, and to be able to breathe without oxygen mask.
> 
> I'm mostly exhausted, but am counting my blessing to be able to be with family and also take care of my toddler until she reaches the age to enter childcare. Then I'll head back to work at AI Singapore.

Regular rest (I know I saw "toddler," and know that means, "impossible") and clean water (I prefer RO) can help immensely with energy. 

Information relative to this pandemic is flooded with misinformation and misdirection where I am (US). I'm glad your mother is somewhere that has the capacity to handle what she's going through. 

Thanks for tagging me with the update. I was worried about the length of time it had been since you last posted. Glad to see that she has improved, and that you're tired but grateful for the silver linings as they manifest.

**Comment by anksankur** [comment]
> Hi @oualidn7 @fuzaro @rafskov @pauljohnfox thanks for dropping a message, I really appreciate it.
> 
> Wow, it has been 4 months.. Mum has transitted to general ward, and seems to be dialysis-free now. We are praying for her kidney function to maintain, and to be able to breathe without oxygen mask.
> 
> I'm mostly exhausted, but am counting my blessing to be able to be with family and also take care of my toddler until she reaches the age to enter childcare. Then I'll head back to work at AI Singapore.

Hope your mom is better now,Ken.Best wishes.

**Comment by kensoh** [comment]
Hi RPA for Python users, 🅗🅐🅟🅟🅨 ②⓪②①!

It has been many months since I last worked on this personal side project. Just want to shout out that this project is very much alive, and I hope to look into the 30+ issues raised since July (and bug-fixes if required) soon when I get a moment again.

Below are details if you are interested, why I'm away past few months and why I believe my darkest days are over 🍀🤞🏻🙏🏻

---

Since May last year, I've stopped all work to juggle with [my mum's critical illness above](https://github.com/tebelorg/RPA-Python/issues/144), taking care of my toddler, and doing housework chores like cleaning and cooking. TBH, my wife and I felt that taking care of a baby and giving her the best nutrition (nursing and home-cooked organic food) and care requires at least 1.5 full-time headcount. There is no life to speak of, other than trying to stay afloat with baby care needs, and keep finding alternative ways to treat my mum, where there're some wins.

Eg importing soy-based tube feeds from Netherlands / Germany / Australia --> after switching away from milk-based tube feeds for 2 weeks she stops requiring dialysis. She started requiring 3x a week dialysis for more than 3 months, after admitting to the hospital. Also, applying herbs like mashed garlic or diced onions to her feet somehow consistently improve her breathing and lung function. Have to keep trying alternative healing methods when her recovery keeps hitting plateau in hospital.

She has been discharged from the hospital a month ago and recovering at home, though very slowly. A longterm medication seems to be one of the contributing causes for her condition. After stopping it, there seems to be improvement to her breathing capacity. She is still bed-bound, but over the course of the year I hope she'll regain strength to move about, and hopefully lungs to recover sufficiently to not require oxygen tank support through a tracheal mask (a hole through her throat).

---

Since November, I've gone back to work at [AI Singapore](https://www.aisingapore.org), a government-funded programme to accelerate AI in Singapore. I'm with Product Engineering team, focusing on RPA and [TagUI (open-source RPA tool)](https://github.com/kelaberetiv/TagUI), which RPA for Python is a wrapper for. There are pretty interesting stuffs on [TagUI 2021 roadmap](https://www.linkedin.com/posts/kensoh_tagui-rpa-v614-release-just-made-it-in-time-activity-6747502665758072833-6YUu). 2020 was the year which my family was going to move to the UK and work, after I receive its tier 1 exceptional tech talent visa. Then everything changed with the virus outbreak, and my mum's illness. The best thing that happened to me in 2020 was being able to go back to work at AI Singapore - I'm grateful.

I've just hired someone to help with housework, cooking and supporting my wife with baby care. I hope this new arrangement will allow me to deliver on my work at AI Singapore, and soon be able to reply countless friends' messages which circumstances leave me with no choice but go missing for months, and then look into the issues raised here!

Eg of the cost of best-in-class nutrition standard researched and set by wife. Baby just felt restless and broke her plate. Even Corelle's break-resistant plate breaks. Why not use plastic plate? Cuz plastic can leech toxins to the food on it. Why not use Silicon plate? Cuz there is no white silicon materials for plate. Why white? Cuz that helps to improve visibility of the food instead of contrasting against some pink colour or some other colours. I would be spending much time clearing up the mess of 70 over pieces of plate fragments, while baby gets nursed to calm down, if not for our newfound helper.

That's why I feel like my darkest days are over and now seeing glimpses of light at the end of the tunnel.

---

My vision for RPA for Python remains - I would like it to be the ***go-to Python package for RPA***.

A best-in-class tool in Python users toolbox which you combine with other cool packages to build amazing stuffs.
